### PR TITLE
Fix JS regression dropdown label

### LINKED ISSUE #5784: [BUG] Uncaught Typeerror when open the "remove" dropdown menu of labels
**To Reproduce**
Open the "remove" dropdown menu of a label in the feed menu

grafik

grafik

**Environment information (please complete the following information):**
- Browser: Firefox 119
- FreshRSS version: 1.22.1

### PR DESCRIPTION
**Pull request by Alkarex**
#fix https://github.com/FreshRSS/FreshRSS/issues/5784

### CODE REVIEW

**Review by Frenzie — APPROVED** [review]

File: p/scripts/main.js [hunk 1]
```diff
@@ -863,8 +863,11 @@ function init_column_categories() {
 				a.href = '#dropdown-' + id;
 				div.querySelector('.dropdown-target').id = 'dropdown-' + id;
 				div.insertAdjacentHTML('beforeend', template);
-				if (feed_web.length < 1) {
-					div.querySelector('.item.link.website').remove();
+				if (feed_web == '') {
+					const website = div.querySelector('.item.link.website');
```
Comment (Alkarex) [inline p/scripts/main.js:867]: I do not dare using *Optional chaining operator* `?.` https://caniuse.com/?search=%3F%2E just yet :-)

Comment (Frenzie) [reply p/scripts/main.js:867]: That wouldn't work here anyway. Or in any case not if you meant `feed_web?.length`. `feed_web` would need to exist, but then `feed_web?.lala?.blabla?.albalb` wouldn't fail.

What I wrote would work (`feed_web && feed_web.length`), but sticking to a string is probably clearer. Incidentally I only just noticed you didn't write `===` though.

Comment (Alkarex) [reply p/scripts/main.js:867]: I meant `div.querySelector('.item.link.website')?.remove();`

Comment (Frenzie) [reply p/scripts/main.js:867]: That one should work. :-)

**Pull request merged**